store/sqlstore/file_info_store.go

```diff
@@ -593,3 +593,40 @@ func (fs SqlFileInfoStore) Search(paramsList []*model.SearchParams, userId, team
 	list.MakeNonNil()
 	return list, nil
 }
+
+func (fs SqlFileInfoStore) CountAll() (int64, error) {
```

Comment:
Maybe we should include some tests for these two new methods in the store

Reply:
done